java-manta-cli/src/main/java/com/joyent/manta/cli/MantaCLI.java

```diff
@@ -384,7 +384,9 @@ public void run() {
                     outputFileName = MantaUtils.lastItemInPath(filePath);
                 }
                 final MantaHttpHeaders headers = new MantaHttpHeaders();
-                headers.setByteRange(startBytes, endBytes);
+                if (startBytes != null || endBytes != null) {
```

Comment:
It is valid for the `endBytes` to not be specified to indicate that the request should return all bytes starting at `startBytes` offset, but it is not valid to omit the `startBytes` value so the condition should just be `if (startBytes != null)`. The specification for the byte range is [here](https://tools.ietf.org/html/rfc2616#section-14.35.1).

Reply:
Noted.